packages/itwinui-css/backstop/tests/listbox.html

```diff
@@ -0,0 +1,213 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Listbox | iTwinUI</title>
+    <script type="module">
+      import "./assets/theme.js";
+      import "@itwin/itwinui-icons-elements/chevron-left";
+      import "@itwin/itwinui-icons-elements/chevron-right";
+      import "@itwin/itwinui-icons-elements/chevron-up";
+      import "@itwin/itwinui-icons-elements/chevron-down";
+      import "@itwin/itwinui-icons-elements/checkmark";
+      import "@itwin/itwinui-icons-elements/search";
+    </script>
+    <style>
+      @import url("./assets/demo.css") layer(demo);
+      @import url('@itwin/itwinui-variables') layer(variables);
+      @import url("@itwin/itwinui-css/css/all.css") layer(itwinui);
+      section[id^='demo'] {
+        --_iui-listbox-menu-height: 200px;
+        width: auto;
+        margin-bottom: 24px;
+      }
+    </style>
+  </head>
+  <body class="iui-root">
+    <theme-button></theme-button>
+    <h1>Listbox</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <div class="iui-listbox-wrapper">
+
+        <div class="iui-listbox-area">
+          <div class="iui-listbox-label">Options</div>
```

Comment:
I placed `iui-listbox-label` inside of `iui-listbox-area` instead of listing them all at the top. This allowed me to get rid of an empty `<div>` that was placed between the labels and used to correct the formatting. The test images have changed slightly as a result